### PR TITLE
Jira added commands

### PR DESCRIPTION
**Pull request by saharsabban**
Added commands to edit issues and status

### CODE REVIEW

**Comment by asafshen** [comment]
@anara123 I won't have time in this sprint to review this properly

File: Integrations/integration-jira.yml [hunk 1]
```diff
@@ -627,5 +656,41 @@ script:
...
+      description: priorty name , like High/Medium
+    - name: dueDate
+      description: DueDate for the issue, in format of 2018-03-11
+    - name: assignee
+      description: assignee name
+    description: Edit JIRA issue
+  - name: jira-get-status-ids
+    arguments:
+    - name: issueId
+      description: issue id to check available ststuses for
```
Comment (anara123) [inline Integrations/integration-jira.yml]: ststuses -> statuses

File: Integrations/integration-jira.yml [hunk 2]
```diff
@@ -54,7 +53,7 @@ configuration:
   defaultvalue: "10000"
   type: 0
   required: false
-- display: Trust any certificate (unsecure)
+- display: Do not validate server certificate (insecure)
```
Comment (anara123) [inline Integrations/integration-jira.yml]: change this back to `Trust any certificate (unsecure)`

File: Integrations/integration-jira.yml [hunk 3]
```diff
@@ -3,7 +3,6 @@ commonfields:
   version: -1
 name: jira
 display: Atlassian Jira
-fromversion: 2.6.0
```
Comment (anara123) [inline Integrations/integration-jira.yml]: don't delete this line

File: Integrations/integration-jira.yml [hunk 4]
```diff
@@ -380,67 +437,39 @@ script:
...
+        case 'jira-get-status-ids':
+            var url = '/rest/api/2/issue/' + String(args.issueId) + '/transitions'
+            var result = sendRequest('GET', url);
+            var jRes = JSON.parse(result.Body);
+            var md = {}
+            for (i = 0; i < jRes.transitions.length; i++ ) {
+                md[jRes.transitions[i].name] = jRes.transitions[i].id;
+            }
+            md = objToMd(md);
+            return {Type: entryTypes.note, Contents: jRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown};
```
Comment (anara123) [inline Integrations/integration-jira.yml]: change all
`{Type: entryTypes.note, Contents: jRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown}`
to 
```
{
    Type: entryTypes.note, 
    Contents: jRes, 
    ContentsFormat: formats.json, 
    HumanReadable: md, 
    ReadableContentsFormat: formats.markdown
}
```

File: Integrations/integration-jira.yml [hunk 5]
```diff
@@ -380,67 +437,39 @@ script:
...
+        case 'jira-issue-update-status':
+            var url = 'rest/api/2/issue/' + String(args.issueId) + '/transitions?expand=transitions.fields'
+            json = {"transition":{"id":String(args.statusId)}}
+            var result = sendRequest('POST', url, JSON.stringify(json));
+                    if (result.StatusCode == 204) {
```
Comment (anara123) [inline Integrations/integration-jira.yml]: fix the indentation

File: Integrations/integration-jira.yml [hunk 6]
```diff
@@ -627,5 +656,41 @@ script:
...
+    - name: labels
+      description: 'comma separated list of label '
+    - name: priority
+      description: priorty name , like High/Medium
+    - name: dueDate
+      description: DueDate for the issue, in format of 2018-03-11
+    - name: assignee
+      description: assignee name
+    description: Edit JIRA issue
+  - name: jira-get-status-ids
```
Comment (anara123) [inline Integrations/integration-jira.yml]: it seems that this command should have outputs

**Comment by Itay4** [comment]
@saharm1 @GavrielFilippov What is the status of this?

**Comment by saharsabban** [comment]
Implementing some few last changes and fixes will upload tomorrow.

**Pull request merged**